meow/mode.py

```diff
@@ -297,7 +297,8 @@ def invert_mode(mode: Mode) -> Mode:
 def inner_product(mode1: Mode, mode2: Mode) -> float:
     """the inner product of a `Mode` with another `Mode` is uniquely defined."""
     mesh = mode1.mesh
-    cross = mode1.Ex * mode2.Hy.conj() - mode1.Ey * mode2.Hx.conj()
+    #cross = mode1.Ex * mode2.Hy.conj() - mode1.Ey * mode2.Hx.conj()
+    cross = mode1.Ex * mode2.Hy - mode1.Ey * mode2.Hx #attention: removed conjugation
```

Comment:
I think this change is correct. EMEpy implements this the same and it seems like it's also done like this in PB thesis.